seqr/views/utils/permissions_utils.py

```diff
@@ -17,10 +19,14 @@ def user_is_data_manager(user):
 def user_is_pm(user):
     return user.groups.filter(name=PM_USER_GROUP).exists() if PM_USER_GROUP else user.is_superuser
 
+def google_login_required(user):
+    return is_google_authenticated(user)
```

Comment:
this function doesn't do anything, it is essentially just adding a different name to an existing function. It would be clearer if you left it out and instead just changed the decorator to use `is_google_authenticated` directly
```
google_auth_required = user_passes_test(is_google_authenticated, login_url=GOOGLE_LOGIN_REQUIRED_URL)
```

Reply:
Sound good.